File: src/Domain/Entities/TestSuite.cs
namespace Domain.Entities;

public class TestSuite : BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public string? Description { get; set; }

    public Guid ProjectId { get; set; }
    public Project Project { get; set; } = null!;

    public ICollection<TestCase> TestCases { get; set; } = [];
}

File: packages/types/src/test-suite.ts
export interface TestSuite {
  id: string;
  projectId: string;
  name: string;
  description?: string;
  createdAt: string;
  updatedAt?: string;
}

export interface CreateTestSuite {
  name: string;
  description?: string;
}

export interface UpdateTestSuite {
  name: string;
  description?: string;
}
